chef/cookbooks/neutron/recipes/common_config.rb

```diff
@@ -150,7 +150,8 @@
       :rootwrap_bin =>  node[:neutron][:rootwrap],
       :use_namespaces => true,
       :allow_overlapping_ips => neutron[:neutron][:allow_overlapping_ips],
-      :dvr_enabled => neutron[:neutron][:use_dvr]
+      :dvr_enabled => neutron[:neutron][:use_dvr],
+      :network_nodes_count => get_network_nodes.count
```

Comment:
This should really just be: `search_env_filtered(:node, "roles:neutron-network").count`; no need to create a library for that.